site/resources.html

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Resources
+id: resources
+---
+<div class="sub-page">
+    <h3>Here you will find external resources about Contour, such as videos, podcasts, and community articles.</h3>
```

Comment:
@amymanion this should be h1

Reply:
Updated to h2 - there is another h1 on the page.

Reply:
@ccbayer There already is an h1, but this should be an h2. Updated it.